lib/LANraragi/Model/Config.pm

```diff
@@ -168,6 +171,28 @@ sub get_tagrules {
     );
 }
 
+# first_install_actions()
+# Setup tasks for first-time installations. New installs are checked by confirming updated
+# user settings. On first installation, create default 'Favorites' category link it to the bookmark
+# button. Returns 1 if is first-time installation, else 0.
+sub first_install_actions {
+    my $redis = get_redis_config();
+    my $logger = get_logger( "Config", "lanraragi" );
+    unless ( $redis->hexists('LRR_CONFIG', 'htmltitle') ) {
+        $logger->info("First-time installation detected!");
+        $redis->hset('LRR_CONFIG', 'htmltitle', 'LANraragi');
+
+        $logger->debug("Creating first category...");
+        my $default_category_id = LANraragi::Model::Category::create_category("Favorites", "", 0, "");
```

Comment:
```suggestion
        my $default_category_id = LANraragi::Model::Category::create_category("🔖 Favorites", "", 0, "");
```  

Let's add an emoji here so it fits with the other default selectors. 
![image](https://github.com/user-attachments/assets/d8a7953e-a82a-4729-b1c9-489f0eccffa7)
Hopefully it makes it intuitive for new users that the bookmark function is linked to this category by default.